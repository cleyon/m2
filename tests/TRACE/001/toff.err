m2trace:toff.m2:5: 'foo' => 'bar'
m2trace:toff.m2:7: 'foo' => 'bar'
