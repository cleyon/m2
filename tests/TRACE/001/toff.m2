@define foo bar
@tracemode +fl
@traceon foo
Start
@foo@
Middle
@foo@
@traceoff foo
@foo@
Done
