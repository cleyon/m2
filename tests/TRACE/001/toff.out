Start
bar
Middle
bar
bar
Done
